scripts/46-audio-receiver.sh

```diff
@@ -0,0 +1,2 @@
+sed -i '/LIBRESPOT_NAME/c\LIBRESPOT_NAME="Neon Mark 2 - '"$(hostname)"'"' /etc/raspotify/conf
```

Comment:
This should probably be passed as an arg so we can handle the name at image build time